provider/src/main/java/com/cloudera/director/google/compute/GoogleComputeInstanceTemplateConfigurationValidator.java

```diff
@@ -475,7 +476,7 @@ void checkNetwork(Configured configuration,
 
     String networkName = configuration.getConfigurationValue(NETWORK_NAME, localizationContext);
 
-    if (networkName != null) {
+    if (networkName != null && !networkName.equals("default")) {
```

Comment:
Why is this check here? There is special handling below as well in GoogleComputeProvider. My guess is that the assumption is that the default network will alwaus be there, so why spend time validating it or creating a network interface specifically naming it. However, it looks like an organization can opt out of one existing:

https://cloud.google.com/vpc/docs/vpc#default-network

So I think it's better to remove the special handling, but anyone with more knowledge can correct me here.